app/components/blacklight/gallery/document_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render(Blacklight::DocumentComponent.new(classes: 'col', component: :div, document: @document, counter: @counter)) do |component| %>
   <% component.with(:body) do %>
     <div class="thumbnail-container">
-      <%= thumbnail %>
+      <%= method(:thumbnail).arity > 0 ? thumbnail(class: 'img-thumbnail') : thumbnail %>
```

Comment:
![🤷](https://i.pinimg.com/originals/9f/f1/92/9ff19223638eda333d18337ac91fb9b9.gif)

Reply:
Except... it's an optional argument, so the arity will actually be -1 😆

Reply:
Give it the ol' switcheroo?

![reverse that](https://media.tenor.com/images/59b5aeffd6893b1c8181d29483fd7ffc/tenor.gif)